azure-specialized-workloads/sap/recommendations.yaml

```diff
@@ -372,15 +372,15 @@
     - name: ASCS-LB - Central Server Instance
       url: "https://docs.microsoft.com/en-us/azure/advisor/advisor-reference-reliability-recommendations"
 
-- description: DBHANA-Pacemaker (Database Instance) Ensure the Pacemaker cluster has been setup for SAP HANA DB high availability
+- description: Ensure the Pacemaker cluster has been setup for high availability
```

Comment:
This should say - "Ensure the Pacemaker cluster has been setup for SAP HANA DB high availability